app/src/main/java/com/hedvig/app/feature/offer/ChangeDateBottomSheet.kt

```diff
@@ -117,9 +104,12 @@ class ChangeDateBottomSheet : RoundedBottomSheetDialogFragment() {
 
                 isDatePicked = true
 
-                dateText.text = "$dayOfMonth $month $year"
+                dialog?.dateText?.text = "$dayOfMonth $month $year"
                 animateHintMove()
-                dateText.show()
+                dialog?.dateText?.show()
+
+                dialog?.chooseDateButton?.isEnabled = true
+                dialog?.chooseDateButton?.style(R.style.HedvigButton_Purple)
```

Comment:
This is kind of cool!
Do we need to add a library for it though? I guess it could be sufficient just to update the colors, maybe?
The reason I'm a bit skeptical is that annotation processors are kind of heavy to introduce, and if we can avoid it we might want to.